allensdk/brain_observatory/behavior/validation.py

```diff
@@ -1,110 +1,172 @@
 import h5py
```

Comment:
Is it obvious from the git history why this file exists?

This almost feels to me like something that was useful a long time ago but is now not worth the trouble of maintaining.

Reply:
I agree with you. Unsure whether to keep it or delete it. It seems like a skeleton of checks that Dan was investing some time in recently.

Reply:
Yeah, the history shows the same intent as what I'd been doing, but, there's been 2+ years since it had science team contributions. So, it is certainly out of date. And, we know no one was running it.
I see a number of similar validation items in https://github.com/AllenInstitute/visual_behavior_analysis
None of those seem to pull this in directly.
So, it is probably safe to remove.